Cookbook/Technical-Documents/TechnicalDebt.md

```diff
@@ -7,62 +7,148 @@ Technical debt and legacy code has a sad tendency to build up in any software pr
 
 GatewayManager supplies global dependencies to legacy Objective-C code. There was plenty of code in version one and two of the Babylon app that relied heavily on singletons.
 
-| Objective-C view Controllers | Location | Comments |
-| ---------------------------- | ---------| -------- |
-| BillingInformationViewController | Clinical Records | CNSMR-930 |
-| EditMembershipCodeViewController | Clinical Records | CNSMR-934 |
-| MembershipTableViewController | Me Tab | CNSMR-926 |
-| SurveyViewController | Clinical Records | CNSMR-937 |
-| InsuranceViewController | Clinical Records | CNSMR-942 |
-| BBAddAdditionalPatientInformationViewControllerV2 | Clinical Records | Might be dead code but will be deleted as part of CNSMR-947 |
+| Objective-C view Controllers | Location | JIRA Tickets | Comments |
+| ---------------------------- | ---------| -------- | |
+| BillingInformationViewController | Clinical Records | CNSMR-930 | |
+| EditMembershipCodeViewController | Clinical Records | CNSMR-934 | |
+| MembershipTableViewController | Me Tab | CNSMR-926 | |
+| SurveyViewController | Clinical Records | CNSMR-937 | |
+| InsuranceViewController | Clinical Records | CNSMR-942 | |
+| BBAddAdditionalPatientInformationViewControllerV2 | Clinical Records | | Might be dead code but will be deleted as part of CNSMR-947 |
+
```

Comment:
The table is not rendering correctly: 

<img width="903" alt="image" src="https://user-images.githubusercontent.com/949286/61398144-4a4a8e80-a8c3-11e9-9fc1-9ecafcd67e9d.png">

Reply:
Thanks, will fix.